website/docs/guides/1.10.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,45 @@
+
+
+**Changes**
+- [Programmatic API Key (or PAK)](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Programmatic-API-Keys) Resources and Data Sources have been updated to make easier and more intuitive to use ([`mongodbatlas_api_key`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/api_key), [`mongodbatlas_project_api_key`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project_api_key) and [`mongodbatlas_project_ip_access_list_key`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/access_list_api_key)). We have created a dedicated [PAK Upgrade Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/PAK-upgrade-guide-1.10.0) to help you transition smoothly from the previous version which was first released in Terraform Provider for MongoDB Atlas in v1.8.0 to the new version in v1.10.0.
+
+
+**Depreciations:**
+- [`mongodbatlas_data_lake`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/data_lake) have been deprecated. We will proceed with code removal in future release targeting v1.12.0. This has been replaced with [`mongodbatlas_federated_database_instance`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/federated_database_instance) 
+- [`mongodbatlas_privatelink_endpoint_service_adl`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service_adl) have been deprecated. We will proceed with code removal in future release targeting v1.12.0. This has been replaced with [`mongodbatlas_privatelink_endpoint_service_data_federation_online_archive`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service_data_federation_online_archive)  
+-  We have removed source code for [`mongodbatlas_private_ip_mode`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/private_ip_mode), NEW_RELIC and FLOWDOCK in [`mongodbatlas_third_party_integration`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/third_party_integration) resources and data sources, and `mongodbatlas_cloud_provider` resources and datas sources including: [`access`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_access), [`snapshot`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot), [`snapshot_backup_policy`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_backup_policy), and [`snapshot_restore_job`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_provider_snapshot_restore_job). 
```

Comment:
Not blocking, just a suggestion: having this be a bulleted list in the bullet would be much easier read imo.

Reply:
updated